src/devices/Ahtxx/README.md

```diff
@@ -0,0 +1,16 @@
+﻿# Ahtxx
+
+## Summary
+AHTxx temperature and humidity sensor family.
+
+## Device Family
+The binding supports the following types:
+* AHT10 - http://www.aosong.com/en/products-40.html (not tested, yet)
+* AHT20 - http://www.aosong.com/en/products-32.html
+
+## Binding Notes
+
+The AHT-devices can be accessed as an I2C bus device. However, you can use only one device per bus as the address is fix.
+
+
```

Comment:
Maybe add a very simple few lines example on how to use the device? So makes it easy for anyone to use rather than going to the sample?

Reply:
And please don't forget as well the category.txt file!

Reply:
Yep. ;)

Reply:
Category.txt added.